gamemaker.py

```diff
@@ -30,4 +69,4 @@ def logout(self):
         return return_value
 
     def display_menu(self):
-        return "Options\n\nlog out\ndisplay status\n"
+        return "Options\n\ndisplaystatus\naddlandmark [clue] [question] [answer]\ndisplaylandmarks\nremovelandmark [position in landmark list]\nlogout\n"
```

Comment:
If the commands are going to be split by commas now, I feel like we should probably indicate that in the Options so that the user knows that they need to include them